src/pages/workspace/accounting/qbo/export/QuickbooksOutOfPocketExpenseConfigurationPage.tsx

```diff
@@ -50,7 +50,7 @@ function QuickbooksOutOfPocketExpenseConfigurationPage({policy}: WithPolicyProps
                             />
                         </OfflineWithFeedback>
                         {exportEntity === CONST.QUICKBOOKS_EXPORT_ENTITY.VENDOR_BILL && !isLocationEnabled && (
-                            <Text style={[styles.ph5, styles.mutedNormalTextLabel, styles.pt1, styles.pb1]}>{translate('workspace.qbo.exportVendorBillDescription')}</Text>
+                            <Text style={[styles.ph5, styles.mutedNormalTextLabel, styles.pt1, styles.pb2]}>{translate('workspace.qbo.exportVendorBillDescription')}</Text>
```

Comment:
Hmm but why is this a one-off style? I would think hint text below a row is a global pattern we use everywhere.

Reply:
@shawnborton unfortunately nope - we do not have such component in place - or probably i did not find it somewhere...
Most of the hints - use this style:
        mutedTextLabel: {
            color: theme.textSupporting,
            fontSize: variables.fontSizeLabel,
            lineHeight: variables.lineHeightLarge,
        }, 

Which i think was probably good to use everywhere. But based on your comments - that lineHeight should be 16px - in all other places variables.lineHeightLarge = 18px.
Probably we need to have global component just for hint

Reply:
I can just change mutedTextLabel to use 16px. But probably not sure how it will be look like in other places

Reply:
Sounds good, that works for me.